vcd/helpers_test.go

```diff
@@ -0,0 +1,63 @@
+// +build vapp vm ALL functional
+
+package vcd
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
```

Comment:
Personally, I would like to see a general comment about this new file, so its purpose is clear and we know what functions to put here in the future.

Reply:
On a second thought maybe I should name the file as `testcheck_funcs_test.go` as this is what it currently contains. Some custom functions which have `TestCheckFunc`signature and are suitable to use together with Terraform built-in `resource.TestCheck...` functions.